UnityPy/classes/Mesh.py

```diff
@@ -555,57 +558,57 @@ def ProcessData(self):
                 )
         # Fix channel after 2018.3
         version = self.version
-        if version >= (2018, 3):  #
-            self.m_VertexData.FixChannel()
+        #if version >= (2018, 3):  #
+        #    self.m_VertexData.FixChannel()
         if version >= (3, 5):  # 3.5 and up
             self.ReadVertexData()
-        
+
         if version >= (2, 6):  # 2.6.0 and later
             self.DecompressCompressedMesh()
-        
+
         self.GetTriangles()
-    
+
     def ReadVertexData(self):
         version = self.version
         m_VertexData = self.m_VertexData
         m_VertexCount = self.m_VertexCount = m_VertexData.m_VertexCount
-        
+
         for chn, m_Channel in enumerate(m_VertexData.m_Channels):
             if m_Channel.dimension > 0:
                 m_Stream = m_VertexData.m_Streams[m_Channel.stream]
                 channelMask = bin(m_Stream.channelMask)[::-1]
-                if channelMask[chn] == "1":
```

Comment:
```csharp
                var channelMask = new BitArray(new[] { (int)m_Stream.channelMask });
                byte offset = 0;
                for (int i = 0; i < 6; i++)
                {
                    if (channelMask.Get(i))
 ```
The channelMask in AssetStudio "gets" a boolean value.
Since BitArray isn't natively available in python bin is used to convert the mask into bits.

The original .get(i) wouldn't fulfill the requirement if a 0 was returned,
but in your variant, a 0 would do, as "0"  is True in python.

Reply:
Thank you so much for correcting the mistake！！！ I  I confused bin with byte